docs/containers/kubernetes/kubernetes-operator.rst

```diff
@@ -73,6 +76,17 @@ A minimal custom resource for a three-node CrateDB cluster may look like this:
              storageClass: <YOUR-STORAGE-CLASS>
            heapRatio: 0.25
 
+.. NOTE::
+
+   The operator imposes an affinity constraint of 1 CrateDB node per k8s node.
```

Comment:
```suggestion
   The operator imposes an affinity constraint of 1 CrateDB node per Kubernetes node.
```
Looks like we usually use the fully spelled-out version throughout the documentation.